lib/node_modules/@stdlib/lapack/base/dlae2/lib/ndarray.js

```diff
@@ -0,0 +1,52 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2024 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var base = require( './base.js' );
+
+
+// MAIN //
+
+/**
+* Computes the eigenvalues of a 2x2 symmetric matrix using alternative indexing semantics.
+*
+* @type {Function}
```

Comment:
You're missing the `@name` tag.

Reply:
I am not sure what change we need to make here

Reply:
https://github.com/stdlib-js/stdlib/blob/50b38f941b50a3f71b531d965c7312d4c6932f84/lib/node_modules/%40stdlib/array/base/assert/is-data-type/lib/main.js#L32

I see that this is missing from `base/special/sqrt` and `base/special/abs`. Feel free to make a PR for those two packages fixing this.